domdiv/main.py

```diff
@@ -498,6 +506,48 @@ def clean_opts(options):
         # keyword to indicate no options.  Same as --fan without any expansions given
         options.fan = []
 
+    if options.tabs_only and options.label_name is None:
+        # default is Avery 8867
+        options.label_name = "8867"
+
+    options.label = None
+    if options.label_name is not None:
+        # Load the Labels, and look for match
+        labels_db_filepath = os.path.join("card_db", "labels_db.json")
+        with get_resource_stream(labels_db_filepath) as labelfile:
+            label_info = json.loads(labelfile.read().decode('utf-8'))
+        assert label_info, "Could not load label information from database"
+        for label in label_info:
+            if options.label_name.upper() in [n.upper() for n in label['names']]:
+                options.label = label
+                break
+
+        if options.label is None:
+            print("Error: Label not defined")
```

Comment:
I think this should fail/throw an exception rather than subtly printing to stdout.